src/misc/host_locale_macos.cpp

```diff
@@ -36,303 +36,306 @@ CHECK_NARROWING();
-	{  19457, { "ru" }         }, // Russian - QWERTY
+	{  19456, { "ru" }             }, // Russian
+	{  19458, { "ru" }             }, // Russian - PC
+	{  19457, { "ru" }             }, // Russian - QWERTY
 	// Russian (extended standard, QWERTY/national)
-	{ -14457, { "rx", 30013 }  }, // Chuvash
-	{  23978, { "rx", 30011 }  }, // Ingush
-	{  19690, { "rx", 30017 }  }, // Kildin Sámi
+	{ -14457, { "rx", 30013 }      }, // Chuvash
+	{  23978, { "rx", 30011 }      }, // Ingush
+	{  19690, { "rx", 30017 }      }, // Kildin Sámi
 	// Swiss (German, QWERTZ)
-	{  19,    { "sd" }         }, // Swiss German
+	{  19,    { "sd" }             }, // Swiss German
 	// Swiss (French, QWERTZ)
-	{  18,    { "sf" }         }, // Swiss French
+	{  18,    { "sf" }             }, // Swiss French
 	// Slovak (QWERTZ)
-	{ -11013, { "sk" }         }, // Slovak
+	{ -11013, { "sk" }             }, // Slovak
 	// Albanian (deadkeys, QWERTZ)
-	{ -31882, { "sq448" }      }, // Albanian
+	{ -31882, { "sq448" }          }, // Albanian
 	// Swedish (QWERTY/ASERTT)
-	{  7,     { "sv" }         }, // Swedish
-	{  224,   { "sv" }         }, // Swedish - Legacy
-	{ -15,    { "sv", 30000 }  }, // Swedish Sámi - PC
-	{ -1203,  { "sv", 30000 }  }, // Lule Sámi (Sweden)
-	{ -1205,  { "sv", 30000 }  }, // Pite Sámi
-	{ -1208,  { "sv", 30000 }  }, // Ume Sámi
+	{  7,     { "sv" }             }, // Swedish
+	{  224,   { "sv" }             }, // Swedish - Legacy
+	{ -15,    { "sv", 30000 }      }, // Swedish Sámi - PC
+	{ -1203,  { "sv", 30000 }      }, // Lule Sámi (Sweden)
+	{ -1205,  { "sv", 30000 }      }, // Pite Sámi
+	{ -1208,  { "sv", 30000 }      }, // Ume Sámi
 	// Tajik (QWERTY/national)
-	{  19460, { "tj" }         }, // Tajik (Cyrillic)
+	{  19460, { "tj" }             }, // Tajik (Cyrillic)
 	// Turkmen (QWERTY/phonetic)
-	{  15228, { "tm" }         }, // Turkmen
+	{  15228, { "tm" }             }, // Turkmen
 	// Turkish (QWERTY)
-	{ -36,    { "tr" }         }, // Turkish Q
-	{ -35,    { "tr" }         }, // Turkish Q - Legacy
+	{ -36,    { "tr" }             }, // Turkish Q
+	{ -35,    { "tr" }             }, // Turkish Q - Legacy
 	// Turkish (non-standard)
-	{ -5482,  { "tr440" }      }, // Turkish F
-	{ -24,    { "tr440" }      }, // Turkish F - Legacy
+	{ -5482,  { "tr440" }          }, // Turkish F
+	{ -24,    { "tr440" }          }, // Turkish F - Legacy
 	// Ukrainian (101-key, QWERTY/national)
-	{ -2354,  { "ua" }         }, // Ukrainian
-	{  19518, { "ua" }         }, // Ukrainian - Legacy  
-	{ -23205, { "ua" }         }, // Ukrainian - QWERTY
+	{ -2354,  { "ua" }             }, // Ukrainian
+	{  19518, { "ua" }             }, // Ukrainian - Legacy  
+	{ -23205, { "ua" }             }, // Ukrainian - QWERTY
 	// Uzbek (QWERTY/national)
-	{  19461, { "uz" }         }, // Uzbek (Cyrillic)
+	{  19461, { "uz" }             }, // Uzbek (Cyrillic)
 	// Vietnamese (QWERTY)
-	{ -31232, { "vi" }         }, // Vietnamese
+	{ -31232, { "vi" }             }, // Vietnamese
```

Comment:
How about splitting out this huge reformat-only part as well into a separate commit?